codeface/R/cluster/graph_comparison.r

```diff
@@ -125,14 +131,17 @@ graph.difference <- function(g1,g2, weighted=FALSE) {
 
 
 ## Compare the results of the tag and non tag based graphs
-graph.comparison <- function(g.1, g.2) {
+graph.comparison <- function(g.1, g.2, weighted=FALSE, symmetric=FALSE) {
   ## Normalize graphs to have binary edge weight
   E(g.1)$weight <- ceiling( scale.data(E(g.1)$weight, 0, 1) )
   E(g.2)$weight <- ceiling( scale.data(E(g.2)$weight, 0, 1) )
 
-  intersectNames <- intersect(V(g.1)$Id, V(g.2)$Id)
-  idx.1 <- match(intersectNames, V(g.1)$Id)
-  idx.2 <- match(intersectNames, V(g.2)$Id)
```

Comment:
You removed the `Id` column so I think your last fix (8ad7f04c28bdbe0b70a35d302dff96403bc3a692) will still fail here.